Package.swift

```diff
@@ -34,7 +34,8 @@ let package = Package(
       dependencies: [
         "CasePaths",
         "CombineSchedulers",
-      ]
+      ],
+      swiftSettings: [.define("DEBUG")]
```

Comment:
Won't this set `DEBUG` for all builds, including release ones?

Reply:
It appears it does set it for all builds. I missed that